DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,7 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
+                ->scalarNode('default_formatter')->end() // TODO: make it required when the major version is changed
```

Comment:
Sorry my question is why there and not in the config before.


```php
        $supportedFormaters = array('foo', 'bar');
```
```php
            ->scalarNode('default_formatter')
                ->info(sprintf(
                    'These following formatters are supported: %s',
                    implode(', ', $supportedFormatters)
                ))
                ->validate()
                    ->ifNotInArray($supportedFormaters)
                    ->thenInvalid('The formatter "%s" is not supported. Please choose one of ('.implode(', ', $supportedDrivers).')')
                ->end()
            ->end()
```

the benefit ist that when we use ```config:dump-reference``` we have the info right there.